main/services/playlists.js

```diff
@@ -94,6 +94,39 @@ module.exports = {
     return saved
   },
 
+  /**
```

Comment:
The drawback of using a new Model class is the necessity of having new service methods, since you can not reuse the ones from playlist service.

I'll suggest to enrich the tracks service with:
```js
getRecent() {
  logger.debug(`fetch recent tracks list`)
  const recent = await builtinListsModel.getById(BuiltinLists.RECENT)
  // I don't use SQL joins on Mélodie. Test proved their are less performing that application joins on Album and Artist list pages. That is why we need to manually fetch tracks in this situation.
  const tracks = await tracksModel.getByIds(recent.trackIds)
  recent.tracks = sorters.rank(recent, tracks)
  return recent 
},

addRecent(trackId) {
  logger.debug({ trackId }, `prepend new track to recent list`)
  const recent = await builtinListsModel.getById(BuiltinLists.RECENT)

  const { saved: [saved] } = await playlistsModel.save({
    ...recent,
    // there is no need to remove duplicates any more. AbstractTrackList ensure their track ids uniqueness, and the first wins. 
    // maybe we should consider capping to a given limit, like the 50 or 100 more recent tracks). Could be further PR.
    trackIds: [trackId, ...recent.trackIds]
  })
  // let's enrich with tracks here as well, so UI has all the data it needs 
  const tracks = await tracksModel.getByIds(recent.trackIds)
  recent.tracks = sorters.rank(recent, tracks)
  // we must broadcast a different event to not overflow with playlist changes
  broadcast('recent-list-change`, saved) 
  return saved
}
```